crates/bevy_transform/Cargo.toml

```diff
@@ -8,6 +8,9 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+trace = []
```

Comment:
Oops, this must be some case of auto-merge failure. The feature should be on `bevy_hierarchy`.

Reply:
🤦 https://github.com/bevyengine/bevy/pull/4477